### PR TITLE
Updated copy to voting section

### LINKED ISSUE #2099: Updates to voting section
Based on user testing & feedback at MozFest https://github.com/mozilla/foundation.mozilla.org/issues/2072 and the [comp](https://redpen.io/bcd47ecb6d670654da).

- [x] Remove `Majority of voters think it is [creepy]` copy from both questions
- [x] Update copy first question to: `How creepy do you think this is?`

image

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #2099

### CODE REVIEW

File: source/js/buyers-guide/components/creep-vote/creep-vote.jsx [hunk 1]
```diff
@@ -117,15 +117,13 @@ export default class CreepVote extends React.Component {
...
   renderVoteAsk() {
-    let creepJudgement = CREEPINESS_LABELS[this.state.majority.creepiness].toLowerCase();
     let confJudgement = this.state.majority.confidence ? `likely` : `not likely`;
 
     return (<form method="post" id="creep-vote" onSubmit={evt => this.submitVote(evt)}>
       <div className="row mb-5">
         <div className="col-12 col-md-6">
           <div className="mb-4 text-center">
-            <h3 className="h5-heading mb-2">How creepy is this product?</h3>
-            <p className="help-text">Majority of voters think it is {creepJudgement}</p>
```
Comment (alanmoo) [inline source/js/buyers-guide/components/creep-vote/creep-vote.jsx]: I think the original ticket asks to remove line 131 as well.

**Comment by beccaklam** [comment]
Hmm, this still appears broken. Do I just need to wait or ... ? @alanmoo @cadecairos

**Comment by alanmoo** [comment]
@beccaklam it works now

**Comment by beccaklam** [comment]
This looks good to go. Only weird thing I noticed is that in chrome we're missing the bottom gradient bar. This is true on the live site too.
**Chrome:**
screen shot 2018-11-12 at 3 46 13 pm
**Comp/Firefox/Safari:**
image
CC: @kristinashu

**Comment by kristinashu** [comment]
Oh good catch! Gradient is there on staging and prod but not here. @mmmavis any idea why?

**Comment by kristinashu** [comment]
Oh weird, I actually see it now?!?

**Comment by alanmoo** [comment]
I think it's a Chrome specific bug that's existed for a while

**Pull request merged**